responses/en/HassRespond.yaml

```diff
@@ -0,0 +1,9 @@
+language: en
+responses:
+  intents:
+    HassRespond:
+      default: "Hello from Home Assistant."
+      listening: "No, I only record when you speak the wake word."
+      data: "Your data is sent to your Home Assistant server."
+      commands: "To learn what you can ask, visit voice.home-assistant.io."
```

Comment:
@mrdarrengriffin That change seems wrong. I brings one to developer docs, not end user docs.

Reply:
> @mrdarrengriffin That change seems wrong. I brings one to developer docs, not end user docs.
> 
> In that regard, it would have probably made more sense to leave a comment instead of committing on this branch.

Totally right, thanks for that.

Do you think https://www.home-assistant.io/voice_control/ is more suitable or is there another URL you have in mind?